tests/examples/CMakeLists.txt

```diff
@@ -216,17 +214,18 @@ function(run_test test_path namespace escaped_content)
 
     # If the image has been created, then add it.
     if(EXISTS "${IMAGE_PATH}.svg")
-        escape_string(
-            "![Usage example](../images/AUTOGEN${namespace}_${TEST_FILE_NAME}.svg)\n"
-            "${TEST_DOC_CONTENT}" TEST_DOC_CONTENT ""
-        )
+        set(OUTPUT_IMAGE_PATH "${IMAGE_PATH}.svg")
     elseif(EXISTS "${IMAGE_PATH}.png")
-        escape_string(
-            "![Usage example](../images/AUTOGEN${namespace}_${TEST_FILE_NAME}.png)\n"
-            "${TEST_DOC_CONTENT}" TEST_DOC_CONTENT ""
-        )
+        set(OUTPUT_IMAGE_PATH "${IMAGE_PATH}.png")
+    else()
+        set(OUTPUT_IMAGE_PATH "")
     endif()
 
+    escape_string(
+        "![Usage example](../images/AUTOGEN${namespace}_${TEST_FILE_NAME}.svg)\n"
+        "${TEST_DOC_CONTENT}" TEST_DOC_CONTENT ""
+    )
```

Comment:
The hardcoded `.svg` seems wrong and doesn't this now also add `Usage examples` images to tests that do not actually produce an image?